db/migrate/20230117193352_change_column_in_table.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeColumnInTable < ActiveRecord::Migration[7.0]
+  def change
+    rename_column :comments, :user_id, :post_id
```

Comment:
- [ ] You have some migration error, kindly check into this 
![image](https://user-images.githubusercontent.com/62705947/213016323-570c1875-b7a3-4270-a771-f879d145b786.png)

Reply:
i didn't add date from luxon. and i removed the change column files so i think it should work. thanks for the review.